### ISSUE TITLE
Replace local "util" dependency

### ISSUE THREAD

**Issue by fwcd** [Xerus2000/monsterutilities#18]
Currently this project depends on the ["util" project](https://github.com/Xerus2000/util), which has to be present in the parent directory of the cloned repository. Generally, it would be better style though to introduce "util" as a remote dependency because this project could be built independently using a single command then.

Large projects solve this by deploying their project dependencies to central repositories (like Maven Central), but this would require some additional effort. Instead, I propose to solve this issue using [jitpack.io](https://jitpack.io), which can be used to conveniently depend on GitHub repositories with a single line (once the repository is added):

```gradle
allprojects {
    repositories {
        ...
        maven { url 'https://jitpack.io' }
    }
}
...
dependencies {
    implementation 'com.github.Xerus2000:util:SOME-COMMIT-ID'
}
```

(The code above needs to be adjusted to use the Kotlin DSL, but you get the idea)

**Comment by fwcd** [comment]
This issue is can be closed once #19 is merged.

**Comment by defvs** [comment]
It is great, issue being that if I need to edit util (themes and icons lies there), I still need to clone it. Right ?

**Comment by fwcd** [comment]
@devfs That should be fine as long as you do not make larger changes to the buildfiles (thus causing merge conflicts later).

**Comment by fwcd** [comment]
Implemented with #19
